src/providers/explore.tsx

```diff
@@ -204,12 +203,8 @@ export const ExploreProvider = ({ children, state, explorePathPrefix = '#/explor
     }
   }, [explorePathPrefix, helia])
 
-  if (helia == null) {
-    return <Loader color='dark' />
-  }
-
   return (
-    <ExploreContext.Provider value={{ exploreState, explorePathPrefix, isLoading, doExploreLink, doExploreUserProvidedPath, doUploadUserProvidedCar, setExplorePath }} key={path}>
+    <ExploreContext.Provider value={{ exploreState, explorePathPrefix, isLoading, doExploreLink, doExploreUserProvidedPath, doUploadUserProvidedCar, setExplorePath }}>
```

Comment:
having the key here was causing a double flash when clicking around.. I originally added it because webui wasn't getting updates.. but the setExplorePath resolves that I believe.